test/components/TagsField_test.js

```diff
@@ -0,0 +1,74 @@
+import { expect } from "chai";
+import sinon from "sinon";
+import { Simulate } from "react-addons-test-utils";
+
+import { createSandbox, createComponent } from "../test_utils";
+import TagsField from "../../src/components/TagsField";
+
+
+describe("TagsField component", () => {
+  let sandbox;
+
+  beforeEach(() => {
+    sandbox = createSandbox();
+  });
+
+  afterEach(() => {
+    sandbox.restore();
+  });
+
+  describe("Separator", () => {
+    it("should use \",\" as a default separator", () => {
+      const node = createComponent(TagsField, {
+        schema: {},
+        formData: ["a", "b", "c"],
+      });
+
+      expect(node.querySelector("input").value)
+        .eql("a, b, c");
+    });
+
+    it("should accept and use a custom separator", () => {
```

Comment:
In that case we are separating with ": " will we separate with two spaces if we use space as a separator?